static/html/5xx.html

```diff
@@ -8,34 +8,27 @@
     <!-- NB: Relies on the fact that PipelineCachedStorage also includes
          a copy of the file with no hash in the name.
          This file will not load in dev unless you set DEBUG = False. -->
-    <link href="/static/min/portico.css" rel="stylesheet">
+    <link href="/static/styles/portico.css" rel="stylesheet">
 
     <meta http-equiv="refresh" content="60;URL='/'">
-</head>
 
-<body>
+</head>
 
+<body id="error_page">
+        <!-- TODO: Make nginx 5xx error page customizable -->
+        <!-- This is tricky because it's not served by Django, -->
+        <!-- so we can't use variables -->
 <div class="container">
 
-<div class="navbar">
-    <ul class="nav pull-right">
-        <li><a href="/accounts/login/?next=/">Log in</a></li>
-    </ul>
-</div>
 <div class="row-fluid">
-    <div class="span2"></div>
-    <div class="span10">
-        <div class="title">
-            <a class="title" href="/">Zulip</a>
-            <span class="for_you"> isn't feeling too good.</span>
+    <img src="../images/500art.svg" alt=""/>
+    <div class="errorbox">
+        <div class="errorcontent">
+            <h1 class="lead">Internal server error</h1>
+            <p>The server has encountered some difficulties. Sorry about that!</p>
+            <p>The page will reload soon after service is restored.</p>
```

Comment:
This should mention it's a Zulip server, at least.  I also feel "some difficulties" feels a bit unclear; I think in this case, it's fair to say "This Zulip server is currently experiencing some technical difficulties.".

Maybe the second sentence should read: "The page will reload automatically soon after service is restored."